staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -116,6 +116,7 @@ type AdmissionMetrics struct {
 	controller       *metricSet
 	webhook          *metricSet
 	webhookRejection *metrics.CounterVec
+	webhookFailOpen  *metrics.CounterVec
```

Comment:
Please see my comment in https://github.com/kubernetes/kubernetes/issues/107146#issuecomment-999175974. I wonder if we can achieve the goal by reusing / extending the existing metrics.